<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>471</width>
      <height>411</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>회원가입</string>
     </attribute>
     <widget class="QLineEdit" name="memberid_edit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>96</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>136</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>PASS :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="join_btn">
      <property name="geometry">
       <rect>
        <x>84</x>
        <y>303</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>회원가입</string>
      </property>
     </widget>
     <widget class="QPushButton" name="reset_btn">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>303</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>초기화</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>190</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>PHONE :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="idcheck_btn">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>96</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>ID CHECK</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="address_edit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>216</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="memberpw_edit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>126</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>24</x>
        <y>246</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>AGE :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>16</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>휴먼엑스포</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(82, 137, 255);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>회원 가입</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>216</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>ADDRESS :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="age_edit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>246</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="phone_edit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>186</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>ID :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="name_edit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>156</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>160</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>NAME :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>로그인</string>
     </attribute>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>휴먼엑스포</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(82, 137, 255);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>회원 로그인</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
       </font>
      </property>
      <property name="text">
       <string>아이디 :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="loginid_edit">
      <property name="geometry">
       <rect>
        <x>164</x>
        <y>104</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>87</x>
        <y>140</y>
        <width>56</width>
        <height>12</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
       </font>
      </property>
      <property name="text">
       <string>비밀번호 :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="loginpw_edit">
      <property name="geometry">
       <rect>
        <x>165</x>
        <y>136</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="login_btn">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>190</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>로그인</string>
      </property>
     </widget>
     <widget class="QPushButton" name="reset_btn_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>190</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>초기화</string>
      </property>
     </widget>
     <widget class="QLabel" name="logintext_label">
      <property name="geometry">
       <rect>
        <x>46</x>
        <y>279</y>
        <width>361</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(226, 255, 245)</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>정보조회</string>
     </attribute>
     <widget class="QPushButton" name="search_btn">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>69</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>조회</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>270</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
       </font>
      </property>
      <property name="text">
       <string>회원나이 :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modify_btn">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>310</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>회원정보수정</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="phone_edit_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>190</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="memberpw_edit_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>110</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>51</x>
        <y>150</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
       </font>
      </property>
      <property name="text">
       <string>회원이름 :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>190</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
       </font>
      </property>
      <property name="text">
       <string>전화번호 :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="age_edit_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>270</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="reset_btn_3">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>310</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>초기화</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="address_edit_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>230</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>230</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
       </font>
      </property>
      <property name="text">
       <string>회원주소 :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="memberid_edit_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>70</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
       </font>
      </property>
      <property name="text">
       <string>회원아이디 :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>52</x>
        <y>110</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
       </font>
      </property>
      <property name="text">
       <string>비밀번호 :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="name_edit_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>150</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>휴먼엑스포</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(82, 137, 255);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>회원 정보 조회</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>556</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>memberid_edit</tabstop>
  <tabstop>memberpw_edit</tabstop>
  <tabstop>name_edit</tabstop>
  <tabstop>phone_edit</tabstop>
  <tabstop>address_edit</tabstop>
  <tabstop>age_edit</tabstop>
  <tabstop>join_btn</tabstop>
  <tabstop>reset_btn</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>loginid_edit</tabstop>
  <tabstop>loginpw_edit</tabstop>
  <tabstop>login_btn</tabstop>
  <tabstop>reset_btn_2</tabstop>
  <tabstop>search_btn</tabstop>
  <tabstop>modify_btn</tabstop>
  <tabstop>phone_edit_2</tabstop>
  <tabstop>memberpw_edit_2</tabstop>
  <tabstop>age_edit_2</tabstop>
  <tabstop>reset_btn_3</tabstop>
  <tabstop>address_edit_2</tabstop>
  <tabstop>memberid_edit_2</tabstop>
  <tabstop>name_edit_2</tabstop>
  <tabstop>idcheck_btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
